MHVLib/Types/MHVPlanObjective.h

```diff
@@ -26,10 +26,16 @@
 @property (readwrite, nonatomic, strong) MHVStringNZNW *name;
 @property (readwrite, nonatomic, strong) MHVStringNZNW *descriptionText;
 @property (readwrite, nonatomic, strong) NSString *state;
-@property (readwrite, nonatomic, strong) MHVPlanOutcomeCollection *outcomes;
+@property (readwrite, nonatomic, strong) MHVPlanOutcomeList *outcomes;
```

Comment:
Is this 'plan' as in 'Action Plan'? If so, are we supposed to be accessing action plan data using the REST services?